src/components/chat-window/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import classNames from 'classnames';
+import { MessageInput } from './message-input';
+
+require('./styles.scss');
+
+export interface Properties {
+  className?: string;
+  isUserConnected?: boolean;
+  onSubmit: (message: string) => void;
+}
+
+export class ChatWindow extends React.Component<Properties> {
+  onSubmit = (message: string) => {
+    this.props.onSubmit(message);
+  };
+
+  render() {
+    return (
+      <div className={classNames('chat-window__input-wrapper', this.props.className)}>
+        {!this.props.isUserConnected && (
+          <div className='chat-window__input-disabled'>You do not have permission to send messages in this channel</div>
+        )}
```

Comment:
for this task, let's just show or not show the input. we don't need to have a message in the case of not being connected.